index.html

```diff
@@ -125,6 +127,66 @@ <h5><a target="_blank" href="https://github.com/pcengines/apu2-documentation/blo
 
 
         <div class="container">
+
+        <!-- mainline releases v4.8.0.5 -->
+ 
+<!--<div class="break"></div>insert this code when a new update comes out to separate a new version from the current.-->
+
+<div class="smaller-margin" id="mr-14">
+        <h2 class="text-middle">Mainline releases</h2>
+        <!-- h2 tag needs to be moved to the newest version -->
+        <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.8.0.4...v4.8.0.5">v4.8.0.5</a></h4>
+    </div>
+
+    <div class="dimmed-text"><p>Release date: 2018-10-04</p></div>
+
+    <ul>
+        <li>Fixed/added:</li>
+        <ol>
+            <li>Updated sortbootorder to v4.6.11</li>
+            <li>Removed custom libpayload modifications</li>
+            <li>Synced and rebased with coreboot official repo commit b7b085d</li>
+            <li>ECC exclusion range, ECC now works properly</li> 
```

Comment:
@pietrushnic maybe we should link the blog post here. What do You think?
@Moviss FYI

Reply:
@pietrushnic @Moviss ?